cmd/cnbBuild.go

```diff
@@ -263,6 +267,15 @@ func runCnbBuild(config *cnbBuildOptions, telemetryData *telemetry.CustomData, u
 		return errors.New("containerRegistryUrl, containerImageName and containerImageTag must be present")
 	}
 
+	if len(config.CustomTLSCertificateLinks) > 0 {
```

Comment:
we recently added a feature to include the custom tls certs natively into the piper http client 
https://github.com/SAP/jenkins-library/blob/4525c1daa4e65b0e3cedd5c603fa8cd602d58b65/pkg/http/http.go#L253

which can be included as  :
```
piperhttp.ClientOptions{
			TrustedCerts:  config.CustomTLSCertificateLinks,
		}
```

This will make the piperhttp client to trust the certs. I am not sure if you are already using it, but this would be a clean solution to handle custom tls certs

Reply:
I think this is not sufficient in our case, because we're running different [lifecycle](https://github.com/buildpacks/lifecycle) binaries inside a container which need to e.g. push to a (internal) registry using a custom TLS certificate.